vast/vast.cpp

```diff
@@ -77,18 +79,23 @@ int main(int argc, char** argv) {
     return EXIT_FAILURE;
   }
   caf::actor_system sys{cfg};
+  // Get filesystem path to the executable.
+  auto binary = detail::objectpath();
+  if (!binary) {
+    VAST_ERROR_ANON("failed to get program path");
+    return EXIT_FAILURE;
+  }
+  auto vast_share = binary->parent().parent() / "share" / "vast";
   // Load event types.
+  auto schema_dirs = std::vector{vast_share / "schema"};
   using string_list = std::vector<std::string>;
-  auto schema_paths = caf::get_if<string_list>(&cfg, "system.schema-paths");
-  if (schema_paths) {
-    if (auto schema = load_schema(*schema_paths)) {
-      event_types::init(*std::move(schema));
-    } else {
-      VAST_ERROR_ANON("failed to read schema dirs:", to_string(schema.error()));
-      return EXIT_FAILURE;
-    }
+  if (auto user_dirs = caf::get_if<string_list>(&cfg, "system.schema-paths"))
```

Comment:
`schema_dirs` and `user_dirs` don't work well together for my taste. If we have `user_dirs`, then I'm expecting `system_dirs`, `default_dirs`, `program_dirs`, or some sort of other `*_dirs` that describes the shipped dirs.